src/test/groovy/com/money_account_service/services/TransferServiceContractTest.groovy

```diff
@@ -1,44 +1,42 @@
 package com.money_account_service.services
 
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.money_account_service.dtos.request.TopUpRequestDto
 import com.money_account_service.dtos.request.TransferRequestDto
 import com.money_account_service.dtos.response.AuthorizeResponseDto
-import com.money_account_service.dtos.response.TopUpResponseDto
 import com.money_account_service.dtos.response.TransferResponseDto
 import com.money_account_service.dtos.response.TransfersResponseDto
+import com.money_account_service.entities.AccountEntity
 import com.money_account_service.entities.TransferEntity
 import com.money_account_service.repositories.TransferRepository
-import com.money_account_service.utility.UserServiceClient
 import org.spockframework.spring.SpringBean
 import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
-import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.http.HttpHeaders
-import org.springframework.test.context.TestPropertySource
-import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers
-import spock.lang.Specification
 
 import java.time.Clock
+import java.time.Instant
 
 class TransferServiceContractTest extends BaseWeb {
 
     @Autowired
     private TransferRepository transferRepository
+    @SpringBean
+    private Clock clock = Mock(Clock)
```

Comment:
You could use just `MockBean`, I think

Reply:
It is the same, but SpringBean is used in Spock, while MockBean is used Mockito